app/src/main/java/org/fossasia/openevent/general/attendees/ListAttendeeId.kt

```diff
@@ -0,0 +1,11 @@
+package org.fossasia.openevent.general.attendees
+
+import com.github.jasminb.jsonapi.LongIdHandler
+import com.github.jasminb.jsonapi.annotations.Id
+import com.github.jasminb.jsonapi.annotations.Type
+
+@Type("attendee")
+data class ListAttendeeId(
+    @Id(LongIdHandler::class)
```

Comment:
why this? the list cannot be an Id especially of long type

Reply:
This is not used anywhere till now. I am making changes to this currently.